lib/rubocop/cop/style/keyword_arguments_merging.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # When passing an existing hash as keyword arguments, add additional arguments
+      # directly rather than using `merge`.
```

Comment:
You might want to add some explanation here as to why using `merge` is a bad idea. And a matching style guide entry.

Reply:
Added explanation.

> And a matching style guide entry

But it already has an example from a style guide?

Reply:
🤦 My bad!